hack/bump-version.sh

```diff
@@ -22,10 +22,10 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
 fi
 
 # parameters
-OPERATOR_OLD="v1\.1\.11"
-OPERATOR_NEW="v1\.1\.12"
-TIDB_OLD="v4\.0\.10"
-TIDB_NEW="v4\.0\.12"
+OPERATOR_OLD="v1\.1\.12"
+OPERATOR_NEW="v1\.1\.13"
+TIDB_OLD="v4\.0\.12"
```

Comment:
Please also cover the DM version in `examples` and `manifests`, and update to v2.0.4.

Reply:
cf07ec4edd372c1d98b8d6aba11f9b0663743e13